orders/models.py

```diff
@@ -31,4 +29,14 @@ class OrderItemStatus(models.Model) :
     status = models.CharField(max_length=200)
 
     class Meta :
-       db_table = 'order_items_status'
\ No newline at end of file
+       db_table = 'order_items_status'
+
+class Delivery(models.Model) :
```

Comment:
```suggestion
class Delivery(models.Model) :
    receiver_name = ...
```
받는 사람에 대한 정보는 따로 추가하지 않아도 되는 상황인가요?